src/board/v3/nap/nap_common.c

```diff
@@ -119,3 +124,64 @@ static void nap_exti_thread(void *arg)
   }
 }
 
+u32 nap_conf_rd_version(void)
```

Comment:
@jacobmcnamee: `nap_conf_rd_version()` should be function to always check the version. format: major (1 byte), minor (1 byte), build (2 bytes). major represents NAPv3, I'll update minor whenever the register interface changes, and build will be representative of smaller features.